submit-web/src/routes/staff/_staffLayout/projects/$projectId/_projectLayout/submission-packages/$submissionPackageId/_submissionLayout/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { PROJECT_STATUS } from "@/components/registration/addProjects/ProjectCard/constants";
+import { ProjectStatus } from "@/components/registration/addProjects/ProjectStatus";
+import { ContentBox } from "@/components/Shared/ContentBox";
+import { YellowBar } from "@/components/Shared/YellowBar";
+import { Box, Grid, Typography } from "@mui/material";
+import {
+  createFileRoute,
+  Navigate,
+  useNavigate,
+  useParams,
+} from "@tanstack/react-router";
+import { BCDesignTokens } from "epic.theme";
+import { PageGrid } from "@/components/Shared/PageGrid";
+import { InfoBox } from "@/components/Submission/InfoBox";
+import { useGetSubmissionPackage } from "@/hooks/api/usePackages";
+import { useGetAccountProject } from "@/hooks/api/useProjects";
+import { useEffect } from "react";
+import { LoadingButton as Button } from "@/components/Shared/LoadingButton";
+import { PackageStatusChipStack } from "@/components/PackageStatusChip/PackageStatusChipStack";
+import { usePackageTableStore } from "@/components/Submission/packageTableStore";
+
+export const Route = createFileRoute(
+  "/staff/_staffLayout/projects/$projectId/_projectLayout/submission-packages/$submissionPackageId/_submissionLayout/"
+)({
+  component: SubmissionPage,
+});
+
+export default function SubmissionPage() {
+  const { reset } = usePackageTableStore();
+  const { projectId: accountProjectIdParam } = useParams({ strict: false });
+  const accountProjectId = Number(accountProjectIdParam);
+  const { data: accountProject } = useGetAccountProject({
+    accountProjectId,
+  });
+  const { submissionPackageId: submissionPackageIdParam } = useParams({
+    strict: false,
+  });
+  const submissionPackageId = Number(submissionPackageIdParam);
+  const { data: submissionPackage } = useGetSubmissionPackage({
```

Comment:
Since at this point we already fetched these, can we use the queryClient.getQueryData to get the account project and the submission package